### PR TITLE
Add dnd area tags

### PR DESCRIPTION
**Pull request by anthmatic**
Dnd area tags are now available via the hubldoc endpoint, this adds the autogen with those and other updates

### CODE REVIEW

File: snippets/auto_gen/hubl_tags.json [hunk 1]
```diff
@@ -484,9 +512,9 @@
     },
     "video_player": {
         "body": [
-            "{% video_player \"my_video_player\"\n\tplayer_id=${player_id}, \n\ttype=\"${type}\", \n\twidth=${width}, \n\theight=${height}, \n\thide_playlist=${hide_playlist}, \n\tviral_sharing=${viral_sharing}, \n\tembed_button=${embed_button}, \n\tcolor=\"${color}\", \n\tplaylist_color=\"${playlist_color}\", \n\tplay_button_color=\"${play_button_color}\", \n\tstyle=\"${style}\", \n\tconversion_asset=\"${conversion_asset}\"\n%}"
```
Comment (williamspiro) [inline snippets/auto_gen/hubl_tags.json:487]: I wonder what happened to the params and description of this?

File: snippets/auto_gen/hubl_tags.json [hunk 2]
```diff
@@ -79,6 +79,41 @@
         "description": "The cycle tag can be used within a for loop to cycle through a series of string values and print them with each iteration\nParameters:\n- string_to_print(String) A comma separated list of strings to print with each interation. The list will repeat if there are more iterations than string parameter values.\n",
         "prefix": "~cycle"
     },
+    "dnd_area": {
+        "body": [
+            "{% dnd_area \"my_dnd_area\"\n\tname=${name}, \n\tlabel=${label}, \n\tclass=${class}\n%}"
```
Comment (williamspiro) [inline snippets/auto_gen/hubl_tags.json:84]: Some history on non-self closing HubL here https://github.com/HubSpot/HubL-Language-Extension/issues/34, which is why these do not have end tags

File: snippets/auto_gen/hubl_tags.json [hunk 3]
```diff
@@ -6,9 +6,9 @@
     },
     "blog_comments": {
         "body": [
-            "{% blog_comments \"my_blog_comments\"\n\tselect_blog=${select_blog}, \n\tlimit=${limit}\n%}"
+            "{% blog_comments \"my_blog_comments\"\n\tselect_blog=${select_blog}, \n\tlimit=${limit}, \n\tskip_css=${skip_css}\n%}"
```
Comment (gcorne) [inline snippets/auto_gen/hubl_tags.json:9]: This probably should be a separate issue, but I feel like we should be consistent between the styleguide, Design Manager code editor defaults, and this extension when it comes to tabs vs. spaces. Perhaps the extension should also be configurable like the Design Manager is.

File: snippets/auto_gen/hubl_tags.json (hunk 2 above)
Comment (anthmatic) [reply snippets/auto_gen/hubl_tags.json:84]: Working on porting the generation script to JS and will handle closing tags from there

**Comment by anthmatic** [comment]
Closing in favor of #54

**Pull request closed without merging**